app/src/main/java/org/simple/clinic/drugs/search/Drug.kt

```diff
@@ -52,7 +52,8 @@ data class Drug(
 
     @Query("""
       SELECT drugs.*, INSTR(lower(drugs.name), lower(:query)) namePosition FROM Drug drugs
-      WHERE namePosition > 0
+      LEFT JOIN ProtocolDrug PD ON PD.rxNormCode == drugs.rxNormCode
+      WHERE PD.uuid IS NULL AND namePosition > 0
```

Comment:
This will exclude all protocol drugs. Don't we only want to exclude protocol drugs from the _current_ facility protocol instead?

Reply:
I did not know that protocols had a facility link 😅 . Will make that change.